### PR TITLE
Grab common steps and support from go maze tests

### PR DESCRIPTION
**Pull request by kinbiko**

### CODE REVIEW

File: lib/features/support/docker.rb [hunk 1]
```diff
@@ -92,6 +92,32 @@ def run_docker_compose_command(file, command, must_pass=true)
...
+      true
+    rescue Errno::ECONNREFUSED, Errno::EHOSTUNREACH
+      false
+    end
+  end
+rescue Timeout::Error
+  false
+end
+
+def wait_for_port(port)
```
Comment (snmaynard) [inline lib/features/support/docker.rb]: Where is this used?

File: lib/features/steps/request_assertion_steps.rb [hunk 2]
```diff
@@ -216,3 +216,22 @@ def validate_error_reporting_thread(payload_key, payload_value, request_index)
   end
   assert_equal(1, count)
 end
+
+Then(/^I wait to receive a request$/) do
```
Comment (snmaynard) [inline lib/features/steps/request_assertion_steps.rb]: Whats this for?

File: lib/features/support/docker.rb (hunk 1 above)
Comment (snmaynard) [reply lib/features/support/docker.rb]: But why doesnt it have an accompanying step?

Comment (CodeHex) [reply lib/features/support/docker.rb]: I think it just got missed, made a PR with the corresponding step https://github.com/bugsnag/maze-runner/pull/49

**Pull request merged**